### PR TITLE
[MIN-93] Create a streamlit sliders for parameters temperature, max_length, and prompt templates

### PR DESCRIPTION
**Pull request by d-lowl**
Add sidebar settings.

When the accept is clicked, hide disclaimer and show settings.
Add sliders for temperature, max_length and prompt template.
Hook them to the LLM querying function.

UI screenshot:
image

TODO:
- [ ] Depends on #60; Merge develop in, after #60 is merged

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -242,18 +248,26 @@ def _get_predictions(
...
+        temperature (float): inference temperature
+        max_length (int): max response length in tokens
 
     Returns:
         str: Summarised text.
     """
     with st.spinner("Loading response..."):
-        payload = _create_payload(messages)
+        payload = _create_payload(messages, temperature, max_length)
+        print(payload)
```
Comment (dudeperf3ct) [inline app/app.py]: do we want to log this instead using `logging.info`?

**Review by dudeperf3ct — APPROVED** [review]
LGTM

**Review by JonoCX — APPROVED** [review]
LGTM 🦕

File: app/app.py (hunk 1 above)
Comment (d-lowl) [reply app/app.py]: Yep, fair

**Review by dudeperf3ct — APPROVED** [review]

**Pull request merged**